tests/utils/siteBuilder.js

```diff
@@ -22,74 +33,75 @@ const createSiteBuilder = ({ siteName }) => {
       const dest = path.join(directory, pathPrefix, 'netlify.toml')
       const content = toToml(config, { space: 2 })
       tasks.push(async () => {
-        await fs.ensureFile(dest)
+        await ensureDir(path.dirname(dest))
```

Comment:
We only need to ensure that the directory exists, as `writeFile` already creates the file if it does not exist (default mode is 'w').